apis/v1alpha1/cluster_fentry_program_types.go

```diff
@@ -17,20 +17,28 @@ limitations under the License.
 // All fields are required unless explicitly marked optional
 package v1alpha1
 
-// ClFentryProgramInfo defines the Fentry program details
 type ClFentryProgramInfo struct {
 	ClFentryLoadInfo `json:",inline"`
-	// Whether the program should be attached to the function.
+	// The hook point for a FEntry program is a Linux kernel function. Unlike a
+	// KProbe program, the FEntry function name is required at load time instead
+	// of attach time. Therefore, the FEntry function is not part of links.
+	//
+	// links is optional, but instead of being a list of hook points like other
+	// eBPF program types, links for FEntry is a list of one, which indicates
+	// whether the FEntry program should be attached or not. The FEntry program is
+	// loaded in kernel memory when the BPF Application CRD is created and the
+	// selected Kubernetes nodes are active. The FEntry program will not be
+	// triggered until the program has also been attached. To attach, add an entry
+	// to links with mode set to Attach. To detach, remove the link entry.
 	// +optional
```

Comment:
Suggestion:
Unlike other eBPF program types, an FEntry program must be provided
with the target function at load time. The links field is optional,
but unlike other program types where it represents a list of hook
points, for FEntry programs it contains at most one entry that
determines whether the program should be attached to the specified
function. To attach the program, add an entry to links with mode set
to Attach. To detach it, remove the entry from links.

Similar for FExit.